x-pack/plugins/cases/docs/openapi/bundled.json

```diff
@@ -31,7 +31,8 @@
   "paths": {
     "/api/cases": {
       "post": {
-        "description": "Creates a case. You must have all privileges for the **Cases** feature in the **Management**, **Observability**, or **Security** section of the Kibana feature privileges, depending on the owner of the case you're creating.\n",
+        "summary": "Creates a case.",
```

Comment:
Just curious, are these auto generated by the `paths` yaml files?

Reply:
For now, they're generated using the npx commands in https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/docs/openapi/README.md